components/calendar/src/error.rs

```diff
@@ -56,6 +56,9 @@ pub enum CalendarError {
     /// An error originating inside of the [data provider](icu_provider).
     #[displaydoc("{0}")]
     Data(DataError),
+    #[displaydoc("The depression angle of angle alpha cannot be reached")]
+    /// The depression angle of the sun is unreachable
+    DepressionAngleUnreachable,
```

Comment:
issue: I don't think this should be an externally exposed error. We should use overflow/underflow or just `()`

Reply:
the enum variant can now be removed